beta/src/styles/sandpack.css

```diff
@@ -90,6 +90,9 @@ html.dark .sp-tabs .sp-tab-button[data-active='true'] {
 .sp-code-editor .cm-errorLine {
   background-color: rgba(255, 107, 99, 0.1);
   position: relative;
+  padding-right: 26px !important;
+  display: inline-block;
```

Comment:
I don't think this display is necessary, this element is already `block` and the following property (min-width) is a consequence of the `display: inline-block`. So, the padding should be more than enough to have the same result.